js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
Why not adding this as a class method, in class body? Can Babel handle `public static` methods?